cli/commands/install.ts

```diff
@@ -0,0 +1,36 @@
+import { runCommand } from "../lib/runner.js";
+
+const directories = [
+  "./tests/cypress",
+  "./nightwatch",
+  "./services/app-api",
+  "./services/ui",
+  "./services/database",
+  "./services/stream-functions",
+  "./services/ui-auth",
+  "./services/ui-src",
+  "./services/ui-waf-log-assoc",
+  "./services/ui-waflog-s3-bucket",
+];
+
+export async function installDeps() {
+  await runCommand(
+    "yarn install root",
+    ["yarn", "install", "--frozen-lockfile"],
+    "."
+  );
+
+  for (const dir of directories) {
+    await runCommand(
+      `yarn install ${dir}`,
+      ["yarn", "install", "--frozen-lockfile"],
+      dir
+    );
+  }
+}
+
+export const install = {
+  command: "install",
+  describe: "install all project dependencies",
+  handler: async () => {},
```

Comment:
This is empty because `installDeps` is being run before every valid command, but I still want `./run --install` to be a valid command in case that's all the user wants to do right then.

Reply:
it is `./run install`